tests/integration/terraform/tf_state.cfn/templates/tf_state.yaml

```diff
@@ -0,0 +1,107 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Conditions:
+  BucketNameOmitted: !Or
+    - !Equals
+      - !Ref 'BucketName'
+      - ''
+    - !Equals
+      - !Ref 'BucketName'
+      - undefined
+  TableNameOmitted: !Or
+    - !Equals
+      - !Ref 'TableName'
+      - ''
+    - !Equals
+      - !Ref 'TableName'
+      - undefined
+Description: Create Terraform backend components - 1.19.0
+Outputs:
+  PolicyArn:
+    Description: Managed policy Arn
+    Export:
+      Name: !Sub '${AWS::StackName}-PolicyArn'
+    Value: !Ref 'ManagementPolicy'
+  TerraformLockTableName:
+    Description: Name of DynamoDB table for Terraform locks
+    Export:
+      Name: !Sub '${AWS::StackName}-TerraformLockTableName'
+    Value: !Ref 'TerraformLockTable'
+  TerraformStateBucketArn:
+    Description: Arn of bucket storing Terraform state
+    Export:
+      Name: !Sub '${AWS::StackName}-TerraformStateBucketArn'
+    Value: !GetAtt 'TerraformStateBucket.Arn'
+  TerraformStateBucketName:
+    Description: Name of bucket storing Terraform state
+    Export:
+      Name: !Sub '${AWS::StackName}-TerraformStateBucketName'
+    Value: !Ref 'TerraformStateBucket'
+Parameters:
+  BucketName:
+    Default: ''
+    Description: (optional) Name for the S3 bucket
+    Type: String
+  TableName:
+    Default: ''
+    Description: (optional) Name for the DynamoDB table
+    Type: String
+Resources:
+  ManagementPolicy:
```

Comment:
Let's delete the `ManagementPolicy`  from the template; it's not used in this context and will make later permission scoping easier

Reply:
removed